2024-frontend/src/components/WasteMeasurementSummary.vue

```diff
@@ -56,19 +56,16 @@ const activeAccordion = ref("")
           Saisir une nouvelle évaluation
         </router-link>
       </div>
-      <div class="fr-col-12 fr-col-sm-4 fr-mb-4w">
-        <div v-if="displayMeasurement.isSortedBySource">
+      <div v-if="displayMeasurement.isSortedBySource" class="fr-col-12 fr-col-sm-8 fr-mb-4w fr-grid-row">
+        <div class="fr-col-12 fr-col-sm-6 fr-pr-2w">
```

Comment:
dans #4672 j'ai rajouté un ``fr-grid-row--gutters` qui permet d'avoir du padding entre chaque col. donc pas besoin du padding-right 
```suggestion
        <div class="fr-col-12 fr-col-sm-6">
```

Reply:
merci !